DigitalLearningSolutions.Data/Helpers/FilteringHelper.cs

```diff
@@ -129,7 +131,13 @@ public AppliedFilter(string filterOption)
             {
                 var splitFilter = filterOption.Split(Separator);
                 Group = splitFilter[0];
-                PropertyName = splitFilter[1];
+                // Course Admin Fields and Centre Registration Prompt filters rely on properties of the models
+                // called Answer1, Answer2 etc. We append the prompt text in brackets to the property
+                // name when setting up these filters so that we can check whether they are valid filters for another course etc.
+                // e.g. Answer1(Access Permission)|Answer1(Access Permissions)|FREETEXTBLANKVALUE would not be a valid filter for
+                // a course without that admin field at position 1, but would be able to be used for filtering the models.

```

Comment:
I feel like the example you've given doesn't exactly match what you've just said. You really want to say that it's not a valid filter for a course with a different admin field at position 1. Right?

Reply:
Err, technically both, will try to reword this